canonical-websites/snapcraft.io
# Updated layout for listing page

Fixes https://github.com/canonical-websites/snapcraft.io/issues/1705

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/<snap>/listing
or
- Use the demo
- See the new layout - ensure fields and combinations still work as expected
- :+1:
